apps/aws-notebook/image/Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM ubuntu:22.04
-RUN apt-get update && apt-get install openssh-server  -y && mkdir -p ~/.ssh &&  mkdir -p /run/sshd
-
+RUN apt-get update && apt-get install -y openssh-server python3-pip
+RUN pip install ipykernel
+RUN mkdir -p ~/.ssh
+RUN mkdir -p /run/sshd
```

Comment:
Please merge these into a single `RUN` command to optimize the size of container image.